tests/fireloop/server/component-config.json

```diff
@@ -4,6 +4,15 @@
   },
   "@mean-expert/loopback-component-realtime": {
     "auth": true,
-    "debug": true
+    "debug": false,
```

Comment:
Why is this change part of this pull request? it seems it squashed commits non related to this functionality.. This modifies the history